Translated/Tutorials/tut_006371.csv

```diff
@@ -1,2 +1,2 @@
 Title#0,"""Stellar Gift Trials"""
-text0001#0,"""Stellar Gift Trials task you with investigating a \""Stellar Gift\"" that has fallen from the sky.<br><br>The fallen Stellar Gift will be protected, so to retrieve it you will need to remove that protection."""
+text0001#0,"""Stellar Gift Trials task you with investigating a \""Stellar Gift\"" that has fallen from the sky.<br><br>The fallen Stellar Gift will be protected, so to retrieve it you will need to disengage that protection."""
```

Comment:
That's not the proper way to use that word. Revert this.

Reply:
It's a completely fine way to use the word. Certainly looks in line with what we see in-game - we don't interact with the capsule's protection directly, we clear the enemies and it turns itself off.